capstone/src/main/java/com/google/sps/servlets/ServeImageServlet.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.blobstore.BlobKey;
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Serves the image file corresponding to the given blob key */
+@WebServlet("/get-image")
+public class ServeImageServlet extends HttpServlet {
```

Comment:
Should this be in a specific sub-repo?

Reply:
Yes, I added another sub-folder for this servlet specifically since it is used in multiple cases